conans/util/files.py

```diff
@@ -188,6 +188,13 @@ def save(path, content, only_if_modified=False, encoding="utf-8"):
     """
     try:
         os.makedirs(os.path.dirname(path))
+    except OSError as error:
+        if error.errno == errno.EACCES:
+            raise OSError("Could not create the directory '{}' due to lack of permission."
+                          " Please, check your user write permission.".format(path))
+        elif error.errno == errno.EEXIST:
+            raise OSError("Could not create the directory '{}' because it already exists."
```

Comment:
If it exists it is OK, no need to raise!

Reply:
Yes, after seeing tons of test errors on my local environment, I agree :joy: